dev/media-template/colleges-base.scss

```diff
@@ -139,6 +145,20 @@
     }
   }
 
+  // Wider icon links
+  .icon-link {
+    @media (min-width: $screen-sm-min) {
+      width: 32%;
+    }
+
+    a h3 {
```

Comment:
Is there a class that can be used here instead of using a nested element selector?